tests/test_wave_01.py

```diff
@@ -116,16 +116,17 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     updated_data = watch_movie(janes_data, MOVIE_TITLE_1)
 
     # Assert
-    assert len(updated_data["watchlist"]) is 0
-    assert len(updated_data["watched"]) is 1
+    assert len(updated_data["watchlist"]) == 0
+    assert len(updated_data["watched"]) == 1
     
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
```

Comment:
It looks like the step of adding our own assertions was missed. What could we assert about the data for a specific movie if we wanted to confirm that the movie in the "watched" list holds the information that we expect it to?